### PR TITLE
Add docs for stream concatenation, React-Chatbot, and Stream-Query-Client

### PR DESCRIPTION
**Pull request by cjcenizal**

### CODE REVIEW

File: www/docs/api-reference/search-apis/stream-query.md [hunk 1]
```diff
@@ -59,38 +59,48 @@ document id and document-level metadata.
...
 
-The third response type, which streams until you get the final `done` value, 
-returns the subsequent streamed chunks of the summary. Each response has a 
+The third response type, which streams until you get the final `done` value,
+returns the subsequent streamed chunks of the summary. Each response has a
 `batchQueryResponse` that contains a portion of the `summary` text.
 
+## Combining the Streamed Summary Response
+
+The consuming code must combing the stream's chunks as it receives them. The best method for doing so will depend on the language being used.
```
Comment (pwoznic) [inline www/docs/api-reference/search-apis/stream-query.md:90]: "combine"

Also, lines should be at or under 80 chars

**Review by pwoznic — CHANGES_REQUESTED** [review]
Just one typo and then lines 80 char (ignore the URLs adding to length of line)

**Review by mrderyk — APPROVED** [review]

**Comment by cjcenizal** [comment]
Thanks @pwoznic ! I addressed your comments. Please merge this as soon as possible.

**Review by pwoznic — APPROVED** [review]
LGTM

**Pull request merged**